bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/SatelBindingConfig.java

```diff
@@ -1,146 +1,146 @@
+     *
+     * @param option
+     *            option to check
+     * @return <code>true</code> if option is enabled
+     */
+    public boolean hasOptionEnabled(Options option) {
+        return Boolean.parseBoolean(getOption(option));
+    }
+
+    /**
+     * Returns value of given option.
+     *
+     * @param option
+     *            option to get value for
+     * @return string value or <code>null</code> if option is not present
+     */
+    public String getOption(Options option) {
+        return this.options.get(option.name());
+    }
+
+    /**
+     * Returns string representation of option map.
+     *
+     * @return string as pairs of [name]=[value] separated by comma
+     */
+    public String optionsAsString() {
+        return this.options.toString();
+    }
+
+    /**
+     * Returns initialization state of bound item.
+     *
+     * @return <code>true</code> if bound item has received state update,
+     *         <code>false</code> if it is uninitialized
+     */
+    public boolean isItemInitialized() {
+        return itemInitialized;
+    }
+
+    /**
+     * Notifies that bound item has its state updated.
+     */
+    public void setItemInitialized() {
+        this.itemInitialized = true;
+    }
+
+    /**
+     * Converts data from {@link SatelEvent} to openHAB state of specified item.
+     *
+     * @param item
+     *            an item to get new state for
+     * @param event
+     *            incoming event
+     * @return new item state
+     */
+    public abstract State convertEventToState(Item item, SatelEvent event);
+
+    /**
+     * Converts openHAB command to proper Satel command that changes state of
+     * bound object (output, zone, etc).
+     *
+     * @param command
+     *            openHAB command to convert
+     * @param integraType
+     *            type of connected Integra
+     * @param userCode
+     *            user's password
+     * @return appropriate Satel command
+     */
+    public abstract SatelCommand convertCommand(Command command, IntegraType integraType, String userCode);
+
+    /**
+     * Returns message needed to get current state of bound object.
+     *
+     * @param integraType
+     *            type of connected Integra
+     * @return a command to send
+     */
+    public abstract SatelCommand buildRefreshCommand(IntegraType integraType);
+
+    protected SatelBindingConfig(Map<String, String> options) {
+        this.options = options;
+        this.itemInitialized = false;
+    }
+
+    protected State booleanToState(Item item, boolean value) {
+        if (item instanceof ContactItem) {
+            return value ? OpenClosedType.OPEN : OpenClosedType.CLOSED;
+        } else if (item instanceof SwitchItem) {
+            return value ? OnOffType.ON : OnOffType.OFF;
+        } else if (item instanceof NumberItem) {
+            return value ? DECIMAL_ONE : DecimalType.ZERO;
+        }
+
+        return null;
```

Comment:
In this case `null` is a marker that the method couldn't handle conversion. In such case the code just skip update of the item.